.github/workflows/build.yml

```diff
@@ -279,14 +305,23 @@ jobs:
         run: cargo xtask integration-test
 
   kubernetes-integration-tests:
-    needs: [build, build-go, build-docs]
+    needs: [get-more-space, build, build-go, build-docs, basic-integration-tests]
     runs-on: ubuntu-latest
     env:
       BPFMAN_IMG: quay.io/bpfman/bpfman:int-test
       BPFMAN_AGENT_IMG: quay.io/bpfman/bpfman-agent:int-test
       BPFMAN_OPERATOR_IMG: quay.io/bpfman/bpfman-operator:int-test
       XDP_PASS_PRIVATE_IMAGE_CREDS: ${{ secrets.XDP_PASS_PRIVATE_IMAGE_CREDS }}
     steps:
+      - name: Check disk space
```

Comment:
I don't think we need to do this before building the images

Reply:
I wanted to see how much disk space we were starting `kubernetes-integration-tests` with, and how much the `docker system prune` recovered.  I also wanted to see how much each step was consuming.  I don't think running `df -h` is very expensive and it's good to keep them in there so we can keep track of the usage over time in case we hit the wall again.